proposals/lip-0051.md

```diff
@@ -2246,29 +2246,17 @@ def removeAllTokensSupportFromChainID(chainID: ChainID) -> None:
 
 ```python
 def supportTokenID(tokenID: TokenID) -> None:
-    chainID = getChainID(tokenID)
-    # All tokens are supported.
-    if there exists entry in the supported tokens substore with key == ALL_SUPPORTED_TOKENS_KEY:
-        return
-
-    # Token already supported.
-    if chainID = OWN_CHAIN_ID or tokenID = getTokenIDLSK():
-        return
-
-    # All tokens from chain are supported.
-    if supportedTokens(chainID) exists:
-        if supportedTokens(chainID).supportedTokenIDs == []:
-            return
-
-        add tokenID to supportedTokens(chainID).supportedTokenIDs, maintaining the array in lexicographical order
 
-    else:
-        create an entry in the supported tokens substore with
-        key = chainID
-        value = encode(
-            schema=supportedTokensSchema,
-            object={"supportedTokenIDs": [tokenID]}
-        )
+    if not isTokenSupported(tokenID):
+	    if supportedTokens(chainID) exists:
```

Comment:
I think indentation is wrong here (both in the nested if-else and in the "add tokenID" part..). It should be same indentation as in the beginning of the function and the else-"create an entry..." part.